index.html

```diff
@@ -424,7 +424,14 @@ <h3 class="mr-xl-5 mb-3 quote">&ldquo;Using a common model and code base, the ON
 								</div>
 							</div>
 							<div class="row equalHeight-3 blue-title-columns">
-								<div class="col-12 col-md-12 mb-4 mb-md-0">
+								<div class="col-12 col-md-4 mb-4 mb-md-0">
+									<div class="sponsor-logo pb-4 pb-md-5 mr-xl-5 text-center">
+										<img class="atlas-logo" src="./images/logos/samtec-logo.png" alt="Samtec logo">
```

Comment:
need to update class name

Reply:
removed class named - they are not actually defined anywhere and not needed

Reply:
can be verified at https://prasanthpul.github.io/onnxruntime/